bolt_store.go

```diff
@@ -168,7 +168,8 @@ func (b *BoltStore) StoreLog(log *raft.Log) error {
 
 // StoreLogs is used to store a set of raft logs
 func (b *BoltStore) StoreLogs(logs []*raft.Log) error {
-	defer metrics.MeasureSince([]string{"raft", "boltdb", "storeLogs"}, time.Now())
+	now := time.Now()
+	defer metrics.MeasureSince([]string{"raft", "boltdb", "storeLogs"}, now)
```

Comment:
Any reason not to combine this and the later defer? It might make it a little easier to notice that the two calls are using the same time interval as their basis.

Reply:
That is a good idea and I have moved it into the deferred func.